poe-tasks/manifest-only-connector-tasks.toml

```diff
@@ -69,6 +69,41 @@ help = "Get the language of the connector from its metadata.yaml file. Use with
 cmd = """yq eval '.data.connectorBuildOptions.baseImage' $PWD/metadata.yaml"""
 help = "Get the base image of the connector from its metadata.yaml file."
 
+[tasks.get-local-version]
+cmd = """yq eval '.data.dockerImageTag' $PWD/metadata.yaml"""
+help = "Get the version of the connector from its metadata.yaml file. Use with -qq to get just the version."
+
+[tasks.create-connector-registry-artifacts]
+shell = '''
+set -eu
+VERSION=$(poe -qq get-local-version)
+CONNECTOR_NAME=$(poe -qq get-connector-name)
+
+# Create build directory
+mkdir -p "build/releases/${VERSION}"
+
+# Clear existing contents
+rm -rf "build/releases/${VERSION}"/*
+
+# Copy required files
+REQUIRED_FILES=("metadata.yaml")
+for file in "${REQUIRED_FILES[@]}"; do
+  cp "$file" "build/releases/${VERSION}/"
+done
+
+# Copy optional files if they exist
+OPTIONAL_FILES=("icon.svg" "manifest.yaml" "components.py")
```

Comment:
We can assume icon.svg is always required, and manifest.yaml is required for manifest-only connectors.